Metro-Records/la-metro-councilmatic
# Refactor event detail

## Overview

This continues the template upgrade to bootstrap 5, this time for the event detail page.

Connects #969 and #996

### Demo

![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/ef5594e5-d488-4f17-837a-4b212eec5c38)

---

![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/4f7330b2-5dcd-4cee-8925-e6fa2e6b9afa)

## Testing Instructions

- Check some events' detail pages against their live versions on desktop and mobile window sizes
- Confirm that
  - they look presentable/similar to the original
  - the functionality still works (accordions interact-able when logged in, pdf displaying properly, etc)
